.gitignore

```diff
@@ -7,3 +7,6 @@ inspect.html
 test/cuda
```

Comment:
I believe this diff is already merged. I would squash merge this PR onto the HEAD another so as to not generate conflicts.

Reply:
It has been. Git shows no conflicts with base branch but I'll merge develop to be safe.